package.json

```diff
@@ -22,40 +22,41 @@
   "keywords": [
     "typescript",
     "angular",
-    "angular6",
+    "angular8",
+    "ng",
     "ng6",
     "ngx",
     "embed-video",
     "embed",
-    "typescript",
     "youtube",
     "vimeo",
     "dailymotion",
-    "video"
+    "video",
+    "iframe"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/SamirHodzic/ngx-embed-video/issues"
   },
   "main": "./dist/index.js",
   "peerDependencies": {
-    "@angular/core": "^6.0.0",
```

Comment:
Why remove this & remove common but not core?

Reply:
The ng update command took care of this 🙂.

Reply:
Seems good so 👌